the-example-app.swift/ContentfulService.swift

```diff
@@ -14,38 +14,52 @@ protocol StatefulResource: class {
     var state: ResourceState { get set }
 }
 
+extension Contentful.Locale: Equatable {}
+public func ==(lhs: Contentful.Locale, rhs: Contentful.Locale) -> Bool {
+    return lhs.code == rhs.code && lhs.name == rhs.name && lhs.fallbackLocaleCode == rhs.fallbackLocaleCode && lhs.isDefault == rhs.isDefault
+}
+
+extension Contentful.Locale {
+
+    /// The default locale of this application and of the associated space in Contentful.
```

Comment:
tribble `/`

Reply:
that's the syntax for public docs in swift @axe312ger ;-)